torch/testing/_internal/common_utils.py

```diff
@@ -1414,8 +1417,50 @@ def f_retry(*args, **kwargs):
     return deco_retry
 
 
-# Methods for matrix generation
+# Methods for matrix and tensor generation
+
 # Used in test_autograd.py and test_torch.py
+def make_tensor(size, device: torch.device, dtype: torch.dtype, *,
+                low, high, requires_grad: bool = False) -> torch.Tensor:
+    """Returns a tensor of the specified size on the given device and dtype.
+       The tensors values are between -9 and 9, inclusive, unless low (high)
```

Comment:
Maybe you want to mention here the special case of bool (that ignore these values), unsigned dtypes (that set low to 0) and integer dtypes (that will floor and ceil the given low and high respectively).

Reply:
I added a clarification